tutor/templates/hooks/mysql/init

```diff
@@ -15,5 +15,6 @@ done
 echo "MySQL is up and running"
 
 # edx-platform database
-mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e 'CREATE DATABASE IF NOT EXISTS {{ OPENEDX_MYSQL_DATABASE }};'
-mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e 'GRANT ALL ON {{ OPENEDX_MYSQL_DATABASE }}.* TO "{{ OPENEDX_MYSQL_USERNAME }}"@"%" IDENTIFIED BY "{{ OPENEDX_MYSQL_PASSWORD }}";'
+mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ OPENEDX_MYSQL_DATABASE }};"
+mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE USER IF NOT EXISTS '{{OPENEDX_MYSQL_USERNAME}}' IDENTIFIED BY '{{ OPENEDX_MYSQL_PASSWORD }}';"
```

Comment:
If the user already exists, but the provided password is different, will it be modified? (it must)

Reply:
@regisb based on the documentation and how MySQL works, since the `IF NOT EXISTS ` part of the statement would prevent the rest of the command to execute, as we know, I split the statement into to:

1. The user creation (if not exists), and
2. Password update/creation (if not set)